fetch_errormessage,events-fired,performanceAPI.smooth,fetch_response,0
NetworkError when attempting to fetch resource.,"[{}, 'error']","{'size': {'transferSize': 0, 'decodedBodySize': 0, 'encodedBodySize': 0}, 'timing': {'duration': >0, 'fetchStart': >0, 'redirectEnd': 0, 'redirectStart': 0, 'secureConnectionStart': 0}, 'initiatorType': 'fetch', 'nextHopProtocol': 'http/1.1'}",uncalled,163660
uncalled,"[{}, 'then']","{'size': {'transferSize': 0, 'decodedBodySize': 0, 'encodedBodySize': 0}, 'timing': {'duration': >0, 'fetchStart': >0, 'redirectEnd': 0, 'redirectStart': 0, 'secureConnectionStart': 0}, 'initiatorType': 'fetch', 'nextHopProtocol': 'http/1.1'}","{'ok': False, 'url': '', 'body': None, 'type': 'opaque', 'status': 0, 'headers': {}, 'redirected': False, 'statusText': ''}",163503
NetworkError when attempting to fetch resource.,"[{}, 'error']","{'size': {'transferSize': 0, 'decodedBodySize': 0, 'encodedBodySize': 0}, 'timing': {'duration': >0, 'fetchStart': >0, 'redirectEnd': 0, 'redirectStart': 0, 'secureConnectionStart': 0}, 'initiatorType': 'fetch', 'nextHopProtocol': ''}",uncalled,29939
